### PR TITLE
Feature/daily summary

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description
Daily summary as the RFC #274 accepted was explained
## Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: media/main.js [hunk 1]
```diff
@@ -34,11 +34,35 @@ gitBlame[0].addEventListener("click", (event) => {
...
     switch (message.command) {
       case "user":
         addGHUserInfo(message.data);
         break;
+        case "dailySummary":
+          console.log("dailySummary");
+          console.log(message.data);
+        $("#dailySummary").append(`
+          <div>
+          <h3>Pull Requests To Review:</h3>
```
Comment (baristaGeek) [inline media/main.js:48]: Good template

**Comment by baristaGeek** [comment]
How do I run and test? Please provide quick instructions.

File: media/main.js (hunk 1 above)
Comment (EstebanDalelR) [reply media/main.js]: Should it be extracted?

**Comment by EstebanDalelR** [comment]
> How do I run and test? Please provide quick instructions.

Pull branch, run extension.
If you run on a Repo you have assigned issues, they will show. If you have nothing assigned, the empty texts will come up.

**Review by baristaGeek — APPROVED** [review]

**Pull request merged**